monkey/common/configuration/default_agent_configuration.py

```diff
@@ -204,5 +204,4 @@
 
 
 def build_default_agent_configuration() -> AgentConfiguration:
-    schema = AgentConfigurationSchema()
-    return schema.loads(DEFAULT_AGENT_CONFIGURATION_JSON)
+    return AgentConfiguration.from_json(DEFAULT_AGENT_CONFIGURATION_JSON)
```

Comment:
We should probably specify the defaults in the marshmallow schema. We'll do it with validation, right?

Reply:
Yeah, #2004.